README.md

```diff
@@ -57,19 +57,29 @@ set pure_symbol_git_up_arrow "^"
 set pure_symbol_git_dirty "!"
 set pure_symbol_horizontal_bar "_"
 
-# Change the colors
-set pure_color_blue (set_color "1e00fd")
-set pure_color_cyan (set_color "1e95fd")
-set pure_color_gray (set_color "6c6c6c")
-set pure_color_green (set_color "66ff66")
-set pure_color_normal (set_color "000000")
-set pure_color_red (set_color "f820ff")
-set pure_color_yellow (set_color "1bc8c8")
-
-# Change colors for username and host in SSH
-set pure_username_color $pure_color_yellow
-set pure_host_color $pure_color_green
-set pure_root_color $pure_color_red
+# Change the base colors
+set pure_color_red (set_color red)
+set pure_color_magenta (set_color magenta)
+set pure_color_blue (set_color blue)
+set pure_color_yellow (set_color yellow)
+set pure_color_cyan (set_color cyan)
+set pure_color_gray (set_color 93A1A1)
+set pure_color_white (set_color white)
+set pure_color_normal (set_color normal)
```

Comment:
This is missing from the changes: `set pure_color_green (set_color "66ff66")`

Keeping them in alphabetical order rather than the random order chosen above would also be 👌

Reply:
Thanks for the feedback 🎉 Changing the order of the colors to be alphabetical is a great idea! The green color isn't included because it does not exist any more, as it's no longer used (replaced by the magenta color)

Reply:
I see plenty of _green_ in the original pure theme screenshot:

<img src="https://raw.githubusercontent.com/sindresorhus/pure/master/screenshot.png">

More generally green is a pretty common color (e.g diff outputs) and should 100% remain as a customizable option, I [I'm not the only user](https://github.com/search?l=fish&q=pure_color_green&type=Code) who has a custom value for `pure_color_green`

As I fully customize _pure_ with my _own_ custom color scheme having the ability to continue to be able to customize the color _green_ should remain.

Reply:
The green in the screenshot is not from pure. Pure is only the prompt, i.e. the stuff before what you type in the terminal. What comes after is dependent on other plugins you use. The green seen in that screenshot is caused by the [hyper-snazzy](https://github.com/sindresorhus/hyper-snazzy) plugin.
The only place pure uses green, in the current master branch, is for the prompt symbol ("❯"). This is magenta (as in the screenshot) in the original pure theme. If you would like to change that, you should change the `pure_color_success` I think.

Reply:
The name of the variable `pure_color_success` could perhaps be renamed to `pure_color_prompt_symbol`, and the `pure_color_error` to `pure_color_prompt_symbol_error`, if that makes it more readable/understandable?

Reply:
@ntwb To customise fish colors use `fish_config colors` or set variables https://fishshell.com/docs/current/index.html#variables in your `config.fish`

Reply:
@boyeborg I would also propose to change `pure_color_gray` to `brblack` terminal color. I mean we are using terminal colors for all elements except this one, seems inconsistent to me. What do you think @edouard-lopez?